src/content/en/updates/2019/08/native-file-system.md

```diff
@@ -202,26 +203,27 @@ async function writeFile(fileHandle, contents) {
 }
 ```
 
-Note: There's no guarantee that the contents are written to disk until
-the `close()` method is called.
+To write data to disk, I needed a [`FileSystemWriter`][fs-writer]. Create one by
+calling `createWriter()` on the file handle object. If permission to write
+hasn’t been granted, the browser will prompt the user for permission. If
+permission isn't granted, , `createWriter()` will throw a `DOMException`, and
+the app will not be able to write to the file. In the text editor, these
+`DOMException`s are handled in the [`saveFile()`][text-editor-app-js] method.
```

Comment:
I reworked this section because it covered a chronological sequence  in non-chronological order. I'd ignore the red, focus on the green, and read carefully.